addons/web/static/src/views/kanban/kanban_controller.scss

```
@@ -103,13 +103,29 @@
             font-size: var(--Card-Footer-font-size, 1rem);
         }
 
-        aside.o_kanban_aside_full > * {
-            height: 100%;
-            @include media-breakpoint-up(md) {
-                height: calc(100% + var(--KanbanRecord-padding-v)* 2);
-                margin-top: calc(var(--KanbanRecord-padding-v)* -1);
-                margin-bottom: calc(var(--KanbanRecord-padding-v)* -1);
-                margin-left: calc(var(--KanbanRecord-padding-h)* -1);
+        aside {
+            flex: 0 0 var(--KanbanRecord__image-width);
+
+            img {
+                height: var(--Card-Aside-Image-height, var(--KanbanRecord__image-width));
+            }
+
+            &.o_kanban_aside_full {
+                --Card-Aside-Image-height: 100%;
+
+                flex: 0 0 var(--KanbanRecord__image--fill-width);
```

What do you think? :)
```suggestion
                --KanbanRecord__image-width: var(--KanbanRecord__image--fill-width);
```